.github/workflows/ci.yml

```diff
@@ -45,15 +45,12 @@ jobs:
         env:
           CI: true
 
-      - name: lint
-        run: |
-          yarn lint
-          yarn workspace stats lint
-
       - name: build sdk packages
         run: |
           yarn workspace @growthbook/growthbook build
           yarn workspace @growthbook/growthbook-react build
+          yarn lint
+          yarn workspace stats lint
```

Comment:
Instead of combining these 2 steps, I would keep them separate and just move linting beneath the sdk step.

Reply:
I made the change and it still passes, so that's great we can keep it separate.